app/src/org/commcare/dalvik/activities/EntityDetailActivity.java

```diff
@@ -218,6 +214,10 @@ public void playVideo(String videoRef) {
         DetailCalloutListenerDefaultImpl.playVideo(this, videoRef);
     }
 
+    public void performCallout(CalloutData callout, int id){
+        DetailCalloutListenerDefaultImpl.performCallout(this, callout, id);

```

Comment:
It makes me sad that there isn't a better design pattern for this listener.
We can't make DetailCalloutListenerDefaultImpl implement DetailCalloutListener because the methods have different argument signatures... it just feels wrong. Anyone know of a way to tie DetailCalloutListenerDefaultImpl to the actual DetailCalloutListener interface?